server/graphql/invoice/src/mutations/outbound_return/update_name.rs

```diff
@@ -0,0 +1,381 @@
+use async_graphql::*;
+
+use graphql_core::{
+    simple_generic_errors::{OtherPartyNotASupplier, OtherPartyNotVisible, RecordNotFound},
+    standard_graphql_error::{validate_auth, StandardGraphqlError},
+    ContextExt,
+};
+use graphql_types::types::InvoiceNode;
+use repository::Invoice;
+use service::auth::{Resource, ResourceAccessRequest};
+use service::invoice::outbound_return::update_name::{
+    UpdateOutboundReturnName as ServiceInput, UpdateOutboundReturnNameError as ServiceError,
+};
+
+use crate::mutations::outbound_shipment::error::InvoiceIsNotEditable;
+
+#[derive(InputObject)]
+#[graphql(name = "UpdateOutboundReturnNameInput")]
+pub struct UpdateNameInput {
+    pub id: String,
+    other_party_id: Option<String>,
```

Comment:
Nit - I think if the whole purpose of this mutation is to change the other_party_id, this shouldn't need to be an option? Would save a few checks...

Reply:
Will still need to do all the checks tho? What checks are you referring to? 🤔  only check that would save is the optional match 👀

Reply:
Oh yeah that's all I meant, the option matches through the validate/generate - felt weird to be running a method called update_name with possibly no name to update to?